ui-performance/index.md

```diff
@@ -121,27 +121,51 @@ Gallery example:
 <center>The graphs produced by the performance overlay.<br>The vertical green
 bars represent the current frame.</center><br>
 
-Each graph represents the last 300 frames for the following two threads used
-in every Flutter app:
+Flutter uses several threads to do its work. You have direct control
+over what happens in the UI thread, but no direct access to the other
+threads.
+
+1. UI thread<br>
+   The platform's main thread executes Dart code in the Dart VM.
```

Comment:
This is incorrect. The platforms main thread is different from the UI thread. The engine spawns a separate UI thread to run Dart code. The platforms main thread is unsurprisingly called the "Platform Thread".